apps/builder/app/builder/features/settings-panel/props-section/animation/animation-keyframes.tsx

```diff
@@ -125,21 +131,28 @@ const Keyframe = ({
   onChange: (value: AnimationKeyframe | undefined) => void;
 }) => {
   const ids = useIds(["offset"]);
+  const apiRef = useRef<CssEditorApi>();
 
-  const cssProperties = useMemo(() => {
-    let result = ``;
-    for (const [property, style] of Object.entries(value.styles)) {
-      result = `${result}${toKebabCase(property)}: ${toValue(style)};\n`;
-    }
-    return result;
-  }, [value.styles]);
+  const styleMap: CssStyleMap = useMemo(
+    () =>
+      new Map(
+        Object.keys(value.styles).map((key) => [
+          key as CssProperty,
```

Comment:
Should be transformed to camelCase as `const parsedStyles = parseCss();` now converts to kebab.
If you expect camelCase.
And parsers working with kebab too.
So in existing json is a mix of camel and kebab. (In the future will be kebab)
Also variables should not be camel cased.
@TrySound knows more about.